requirements.txt

```
@@ -27,3 +27,4 @@ pycodestyle==2.3.1
 pylint>=2.2.2
 astroid>=2.1.0
 isort>=4.3.4
+setuptools>=41.2.0
```

```suggestion
setuptools>=20.7.0
```
Ubuntu 19.10 only provides 41.1.0. The `SpecifierSet` feature has been available for 5 years. I just tried with the Ubuntu 16.04 version of setuptools (20.7.0) and it worked fine.

---

okay